addons/digest/views/res_config_settings_views.xml

```diff
@@ -5,23 +5,19 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="base_setup.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@id='contacts_settings']" position="before">
-                <div id="statistics" >
-                    <block title="Statistics" id='statistics_div'>
-                        <setting string="Digest Email" help="Add new users as recipient of a periodic email with key metrics" documentation="/applications/general/digest_emails.html" title="New users are automatically added as recipient of the following digest email." name="digest_email_setting_container">
-                            <field name="digest_emails"/>
-                            <div class="content-group" invisible="not digest_emails">
-                                <div class="mt16">
-                                    <label for="digest_id" class="o_light_label mr8"/>
-                                    <field name="digest_id" class="oe_inline"/>
-                                </div>
-                                <div class="mt8">
-                                    <button type="action" name="%(digest.digest_digest_action)d" string="Configure Digest Emails" icon="oi-arrow-right" class="btn-link"/>
-                                </div>
-                            </div>
-                        </setting>
-                    </block>
-                </div>
+            <xpath expr="//setting[@id='restrict_template_rendering_setting']" position="before">
```

Comment:
I'm not moving code, so I wasn't sure that [MOv] would be appropriate. Maybe on the other commit, but I'm not sure it would fit either, since I'm also adding a few lines?
Either way, I did forget to add the module name. Fixing that in a sec!

Reply:
Actually you are right, it is just an IMP or a REF if we want to be complete :)

Reply:
I... guess the first commit could count as a refactor, then? Should I change it?

Reply:
There is no clear rule, but generally in master everything is an IMP (except REF which is used for more model oriented changes, or MOV is you do only code move). Here I would say IMP (obviously, we are doing bike shedding).

Reply:
Aye.